modules/replicator/src/main/java/org/apache/ignite/internal/replicator/exception/PrimaryReplicaMissException.java

```diff
@@ -28,6 +30,17 @@
 public class PrimaryReplicaMissException extends IgniteInternalException implements ExpectedReplicationException {
     private static final long serialVersionUID = 8755220779942651494L;
 
+    /**
+     * Constructor.
+     *
+     * @param txId Transaction id.
+     * @param expectedEnlistmentConsistencyToken Expected enlistment consistency token, {@code null} if absent.
+     */
+    public PrimaryReplicaMissException(UUID txId, Long expectedEnlistmentConsistencyToken) {
+        super(REPLICA_MISS_ERR, format("The primary replica has changed [txId={}, expectedEnlistmentConsistencyToken={}].",
```

Comment:
Let's also log current lease start time, or basically all information about current lease, if not null.

Reply:
done